Source/WebCore/page/ResizeObserver.cpp

```diff
@@ -42,12 +42,17 @@ namespace WebCore {
 
 Ref<ResizeObserver> ResizeObserver::create(Document& document, Ref<ResizeObserverCallback>&& callback)
 {
-    return adoptRef(*new ResizeObserver(document, WTFMove(callback)));
+    return adoptRef(*new ResizeObserver(document,  { RefPtr<ResizeObserverCallback> { WTFMove(callback) } }));
```

Comment:
Typo: you used 2 spaces after the comma. And shouldn't this work too?

```cpp
return adoptRef(*new ResizeObserver(document, { WTFMove(callback) }));
```

Reply:
Done, and nope, the compiler complains.